README.md

```diff
@@ -24,30 +26,30 @@ and provides a utility that gets tests up and running quickly
     ```
 
   * writes to `testHelperPath`
-    ```coffee
-    chai = require 'chai'
-    sinon = require 'sinon'
-    chai.use require 'sinon-chai'
+    ```js
+    var chai = require('chai');
+    var sinon = require('sinon');
+    chai.use(require('sinon-chai'));
 
-    global.expect = chai.expect
-    global.sinon = sinon
+    global.chai = chai;
```

Comment:
You were the one who added this :smile: 

https://github.com/Originate/mycha/commit/10f2883b10d67f173c7503508be94200e429d10c